rllib/algorithms/dqn/dqn_rainbow_catalog.py

```diff
@@ -0,0 +1,281 @@
+                        "fcnet_weights_initializer"
+                    ],
+                    hidden_layer_weights_initializer_config=self._model_config_dict[
+                        "fcnet_weights_initializer_config"
+                    ],
+                    hidden_layer_bias_initializer=self._model_config_dict[
+                        "fcnet_bias_initializer"
+                    ],
+                    hidden_layer_bias_initializer_config=self._model_config_dict[
+                        "fcnet_bias_initializer_config"
+                    ],
+                    output_layer_activation=self.af_and_vf_encoder_activation,
+                    output_layer_dim=self.latent_dims[0],
+                    output_layer_use_bias=self._model_config_dict[
+                        "output_layer_use_bias"
+                    ],
+                    # TODO (sven, simon): Should these initializers rather the fcnet
+                    # ones?
+                    output_layer_weights_initializer=self._model_config_dict[
+                        "post_fcnet_weights_initializer"
+                    ],
+                    output_layer_weights_initializer_config=self._model_config_dict[
+                        "post_fcnet_weights_initializer_config"
+                    ],
+                    output_layer_bias_initializer=self._model_config_dict[
+                        "post_fcnet_bias_initializer"
+                    ],
+                    output_layer_bias_initializer_config=self._model_config_dict[
+                        "post_fcnet_bias_initializer_config"
+                    ],
+                    std_init=self.std_init,
+                )
+        # Otherwise return the base encoder config chosen by the parent.
+        # This will choose a CNN for 3D Box and LSTM for 'use_lstm=True'.
+        return self._encoder_config()
+
+    def _get_head_config(self, output_layer_dim: int):
+        """Returns a head config.
+
+        Args:
+            output_layer_dim: Integer defining the output layer dimension.
+                This is 1 for the Vf-head and `action_space.n * num_atoms`
+                for the Af(Qf)-head.
+
+        Returns:
+            In case noisy networks should be used a `NoisyMLPHeadConfig`. In
+            the other case a regular `MLPHeadConfig` is returned.
+        """
+        # Define the config to use for the heads.
+        config_cls = NoisyMLPHeadConfig if self.uses_noisy else MLPHeadConfig
+
+        # Define the kwargs to pass in the standard deviation to the noisy network.
+        kwargs = {"std_init": self.std_init} if self.uses_noisy else {}
+
+        # Return the appropriate config.
+        return config_cls(
+            input_dims=self.latent_dims,
+            hidden_layer_dims=self._model_config_dict["post_fcnet_hiddens"],
+            hidden_layer_activation=self.af_and_vf_head_activation,
+            # TODO (simon): No clue where we get this from.
```

Comment:
@sven1977 Where do we actually get these config settings? These are not given in `models.catalog.py`.

Reply:
I think they only exist in the new model config objects, so thus far, they are not really reachable by users going through the traditional Catalog model_config_dict.